grails-app/controllers/org/bbop/apollo/GroupController.groovy

```diff
@@ -55,7 +55,10 @@ class GroupController {
         try {
             log.debug "loadGroups"
             JSONObject dataObject = permissionService.handleInput(request, params)
-
```

Comment:
not sure why we are adding this here if we make the same call down below?   Calling here is fine (and maybe preferable), but I think rechecking for admin later makes little sense and you might wish to remove that code.   Also, is it suffiient to have INSTRUCTOR using GLOBAL_PERMISSION_ENUM instead as an INSTRUCTOR may want to load groups as well? 

Also, might